xds/src/test/java/io/grpc/xds/XdsClientWrapperForServerSdsTestMisc.java

```diff
@@ -55,7 +54,8 @@
   @Before
   public void setUp() throws IOException {
     MockitoAnnotations.initMocks(this);
-    xdsClientWrapperForServerSds = new XdsClientWrapperForServerSds(PORT, xdsClient, null);
+    xdsClientWrapperForServerSds = new XdsClientWrapperForServerSds(PORT);
+    xdsClientWrapperForServerSds.start(xdsClient);
```

Comment:
same here. cleanup the resource.

Reply:
Same as above. There is no real client or a real connection to anything - so nothing to clean up AFAIS

Reply:
done